index1.html

```diff
@@ -5,12 +5,27 @@
     <title>Project: Rock, Paper, Scissors</title>
   </head>
   <body>
+    <div class='c-div__heading'>
+      <h1 class='c-div__heading--h1'>Welcome to my Rock Paper Scissors Game</h1>
+    </div>
+    <div class='c-div__buttons'>
+      <button type='button' value='rock'>Rock</button>
+      <button type='button' value='paper'>Paper</button>
+      <button type='button' value='scissors'>Scissors</button>
+    </div>
+    <div class='c-div__results'>
+      
+    </div>
+
+
+
     <script>
+
       let result
       let playerScore = 0
       let computerScore = 0
 
-      game()
+      computerSelection = computerPlay()
```

Comment:
This will get assigned once when the page loads and won't get called again, try updating line 50 with `playRound(e.target.value, computerPlay(), true)` and see if that works